class/defaults.yml

```diff
@@ -1,8 +1,18 @@
 parameters:
   eventrouter:
     namespace: syn-eventrouter
+
     images:
       eventrouter:
-        registry: "gcr.io"
-        image: "heptio-images/eventrouter"
-        tag: v0.3
+        registry: "registry.redhat.io"
+        image: "openshift-logging/eventrouter-rhel8"
+        tag: v0.4
```

Comment:
We can't use this image as the default, since you need a pull secret to pull from registry.redhat.io.

Reply:
Ah shoot... `isOpenshift` check it is.
Also it doesn't need any special privileges -> no breaking change.